commands/Misc/cat.js

```diff
@@ -12,8 +12,8 @@ module.exports = class extends Command {
 	}
 
 	async run(msg) {
-		const { body } = await snek.get('http://random.cat/meow');
-		return msg.send('I found this cat image. Here you go!', { files: [{ attachment: body.file, name: `cat.${body.file.split('.')[2]}` }] });
+		const { body } = await snek.get("http://random.cat/meow");
+        await msg.channel.send({ files: [{ attachment: body.file, name: `cat.${body.file.split(".")[2]}` }] }).catch(e => msg.channel.send(e));
```

Comment:
You are mixing tabs and spaces, and you **should** return that message.
Remove that useless catch if you're not throwing a custom message (based on the error), if it's about Nomsy in An Idiots Guide, don't listen to him, because this is not how Klasa works.